network/vpc/reference-content/vpc-migration.mdx

```diff
@@ -0,0 +1,133 @@
+
+When VPC goes into GA, you will see the following changes on our devtools and console:
+
+**Scaleway console**
+- The existing **Private Networks** section, available from the main menu (side bar) of the [console](https://console.scaleway.com/), will soon cease to be accessible. Instead, you should select **VPC** in the side bar, and then click on the relevant VPC, to access and manage your Private Networks.
+- The DHCP dashboard for **Public Gateways** will no longer be available, as this functionality moves to Private Networks.
+
+**API**
+- The VPC v1 zoned API (which actually only concerns Private Networks and not VPC as it is now understood) will keep working, but will be deprecated and moved to read only. The VPC v2 API, covering both VPCs and Private Networks with regional scope and DHCP, will be the new default API. Note that on the [VPC API documentation](https://www.scaleway.com/en/developers/api/vpc/v2/), you can select v1 or v2 from the drop down menu at the top to navigate between the two versions.
+- [DHCP reservations via the Public Gateway API](https://www.scaleway.com/en/developers/api/public-gateway/) will be deprecated but still available, to avoid breaking changes. In the future (no ETA), reservations should be available on Private Networks through IPAM.
+
+**CLI**
+- As the CLI does not support multiple versions of an API, new and future versions will drop the VPC v1 API in favour of v2.
+- DHCP reservations via the Public Gateway will be deprecated but still available, to avoid breaking changes. In the future (no ETA), reservations should be available on Private Networks through IPAM.
+
+**Terraform**
+- Currently, a property exists on Private Network resources, to specify whether they are regional or not and allow use of the correct underlying API (VPC v1 or v2). This property will be deprecated once VPC is in GA.
+- DHCP reservations via the Public Gateway will be deprecated but still available, to avoid breaking changes. In the future (no ETA), reservations should be available on Private Networks through IPAM.
+
+## Why are we making these changes?
+
+Security and networking are not easy. Mistakes can be made and details can be forgotten which can lead to conflicts in your configuration or serious security holes in your infrastructure. 
+
+We are aiming to make it as easy as possible for our customers to create a secure infrastructure, with resources reachable in a simple yet safe way, able to communicate with each other without having to be exposed on the public Internet.
+
+Each Scaleway Project will get a default VPC for each region. From there, it will only take a couple of clicks to create a Private Network with built-in DHCP in a VPC. No network configuration will be required, all the defaults are set up and ready to go. We even take care of making sure that each Private Network gets a large enough subnet that it won't overlap with that of any other Private Networks inside the same VPC.
```

Comment:
```suggestion
Each Scaleway Project will get a default VPC for each region. From there, it will only take a couple of clicks to create a Private Network with built-in DHCP in a VPC. No network configuration will be required, all the defaults are set up and ready to go. We even take care of making sure that each Private Network gets a large enough subnet that it will not overlap with that of any other Private Networks inside the same VPC.
```